backend/tests/upload/test_model.py

```diff
@@ -0,0 +1,58 @@
+import json
+import os
+
+import pytest
+from django.core.files.storage import default_storage
+from django.test import RequestFactory
+
+from upload.models import Upload
+from upload.views import file_upload
+
+
+@pytest.fixture()
+def file_sample():
+
+    path = "testfile.txt"
+    f = open(path, "w")
+    f.write("Hello World")
+    f.close()
+
+    post_data = {"file": open(path, "rb")}
+
+    yield (path, post_data)
```

Comment:
This looks like a good solution to clean up the files once you are done with them, I usually just move the tests in a temporary media folder and add a pytest post hook to clean up the folder once all the tests are run. This one seems like an interesting solution